tox.ini

```diff
@@ -16,7 +16,7 @@ commands =
        pytest -n {posargs} -s -v --timeout=300 --reruns=2 --reruns-delay=60 \
               --junitxml=test-reports/junit.xml --log-file=test-reports/logs.txt tests
 
-[testenv:py39]
+[testenv:py10]
```

Comment:
Is this really `py10` or should it be `py310`?

Reply:
Typo, thanks